build.gradle

```diff
@@ -48,7 +37,7 @@ subprojects {
 
     ext.guavaVer  = '26.0-jre'
     ext.asmVer    = '7.0'
-    ext.kotlinVer = '1.3.72'
+    ext.kotlinVer = kotlinVersion
```

Comment:
Do you need this `ext.kotlinVer` now? Can't you use `kotlinVersion` directly instead, assuming you're even still using it at all?

Reply:
I wondered the same actually, I'll take it out.